### PR TITLE
InstructorCoursesEditPageUiTest fails on line 438 #4397

### LINKED ISSUE #4397: InstructorCoursesEditPageUiTest fails on line 438
diff

Probably due to some changes in #4284. @imouto1994

### PR DESCRIPTION
**Pull request by imouto1994**
Fixed #4397

### CODE REVIEW

**Comment by kanghj** [comment]
Good to merge.

File: src/main/webapp/js/common.js [hunk 1]
```diff
@@ -534,6 +534,23 @@ function setStatusMessage(message, error) {
...
 /**
+ * Append a message to the existing status message
+ * @param  message
+ * 
+ */
+function appendStatusMessage(message, error) {
+    if (message.trim() === '') {
+        return;
+    }
+    const currentContent = $(DIV_STATUS_MESSAGE).html();
```
Comment (kanghj) [inline src/main/webapp/js/common.js]: #4442 is failing because of this. @imouto1994, we can just change this to a var right?

Comment (imouto1994) [inline src/main/webapp/js/common.js]: Oops, sorry since I mistakenly used ES6. Will fix it :)

Comment (kanghj) [inline src/main/webapp/js/common.js]: Try to fix it soon because it's a critical issue. :)

**Pull request merged**